FFWEB-1566: Add information about `topic` attribute (#239)
FACT-Finder-Web-Components/documentation: markdown/3.x/en/api/ff-navigation.api.md

| **fetch-time**&nbsp;(Number) (default: 100) | Defines the time between navigation requests in milliseconds. |
| **subscribe**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "true") | Defines if component should automatically subscribe to `navigation` event and use navigation data retrieved from FACT-Finder. |
| **navigationData**&nbsp;(Array) | The data elements used to render the navigation. Each element in the array represents a header menu item with all its sub-items for all cluster levels defined by the `max-fetch` property (see above). If `navigationData` is set by the user, it may contain sub-elements of any level, regardless of `max-fetch`'s value. |

> | **topic**&nbsp;(String) | Defines if component should subscribe to custom topic instead of default one. Please note that in this case, you need to manually dispatch navigation data in correct form. For more information regarding its structure, please refer to **Custom navigation data**  in [Documentation](/api/3.x/ff-navigation#tab=docs) section. |

Reviewer: I think without `in correct form` the explanation is still well comprehensible and has less potential for confusion.
```suggestion
| **topic**&nbsp;(String) | Defines if the component should subscribe to a custom topic instead of the default one. Please note that in this case you need to manually dispatch navigation data. For more information regarding the data's structure, please refer to **Custom navigation data** in the [ff-navigation documentation](/api/3.x/ff-navigation#tab=docs). |
```

Author: changed
